gulpfile.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var gulp = require('gulp');
+
+var testTask = require('./gulp-tasks/karma-unit-test.js');
+
+/**
+ * Runs Karma unit tests
```

Comment:
Add period at the end. Please check this at other position in the file.

Reply:
As discussed, ignoring this comment, as this is not enforced by semistandard.